recipes-debian/libkrb5/libkrb5_debian.bb

```diff
@@ -0,0 +1,119 @@
+# base recipe: meta-openembedded/meta-oe/recipes-connectivity/krb5/krb5_1.17.bb
+# base branch: warrior
+# base commit: a24acf94d48d635eca668ea34598c6e5c857e3f8
+SUMMARY = "Runtime library for the Kerberos v5 API for the main part used by the application and the Kerberos client"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://NOTICE;md5=aff541e7261f1926ac6a2a9a7bbab839"
+
+DEPENDS = "bison-native ncurses util-linux e2fsprogs e2fsprogs-native openssl"
+
+inherit autotools-brokensep binconfig debian-package
+require recipes-debian/sources/krb5.inc
+
+DEBIAN_UNPACK_DIR = "${WORKDIR}/krb5-${PV}"
+
+SRC_URI += "file://0001-aclocal-Add-parameter-to-disable-keyutils-detection.patch;patchdir=src \
+           file://crosscompile_nm.patch;patchdir=src"
+
+EXTRA_OECONF += " --without-tcl --with-system-et --disable-rpath"
+CACHED_CONFIGUREVARS += "krb5_cv_attr_constructor_destructor=yes ac_cv_func_regcomp=yes \
+                  ac_cv_printf_positional=yes ac_cv_file__etc_environment=yes \
+                  ac_cv_file__etc_TIMEZONE=no"
+
+CFLAGS_append = " -fPIC -DDESTRUCTOR_ATTR_WORKS=1 -I${STAGING_INCDIR}/et"
+CFLAGS_append_riscv64 = " -D_REENTRANT -pthread"
+LDFLAGS_append = " -pthread"
+
+# The krb5 source is extracted under src dirctory, so set the confiure file path.
+# To apply debian patches, extracted the krb5 source under src dirctory.
+CONFIGURE_SCRIPT = "${S}/src/configure"
+
+do_configure() {
+    # Change Dirctory where the configure.in exists.
+    cd src
+    gnu-configize --force
+    autoreconf
+    oe_runconf
+}
+
+do_compile_prepend() {
+    # Change Dirctory where the Makefile exists.
+    cd src
+}
+
+do_install() {
```

Comment:
Why do you install all files in one package?
[meta-openembedded's krb5_1.17.bb](https://cgit.openembedded.org/meta-openembedded/tree/meta-oe/recipes-connectivity/krb5/krb5_1.17.bb?h=warrior) and debian provide indipentent packages.

Reply:
I think following there way is better.